src/app/pages/forms-and-validation/phone-number-format/phone-number-format.component.scss

```
@@ -0,0 +1,42 @@
+@import '~@pxblue/colors/palette.scss';
+
+section.mat-typography {
+    padding-top: 40px;
+    max-width: 480px;
+    margin: 0 auto;
+    .phone-number-form {
+        display: flex;
+        .country-code-select {
+            max-width: 112px;
```

```suggestion
            max-width: 135px;
```

Let's give that select arrow some breathing room. It looks like you set the min-width to match the desktop select input design that said "area code". I'd recommend changing it to match the select input width that is in the mobile design.